docs/Containers/Container-Tasks.md

```diff
@@ -0,0 +1,279 @@
+
+A new tab has been added to the project settings page for container
+configurations. Users can create a list of resource configuration
+presets that can be referenced via alias in the size field of their
+container YAML configurations.
+
+![containers.png](../images/container_configuration.png)
+
+Options:
+
+-   Name: The alias for the resource preset. Names must be unique within
+    the list.
+
+-   Memory: The amount of memory (in MiB) to allocate.
+
+-   CPU: The CPU units the container can use. These values are expressed
+    in 'vCPU Units'. 1024 CPU units is the equivalent of 1vCPU.
+
+Users can define as many container configurations as needed, reflecting
+different appropriate resource needs for various tasks.
+
+#### **Task Metadata**
+
+A link to a container task's respective container replaces the typical
+host link.
+
+<img alt="containerized_task_metadata.png" height="400" src="../images/container_metadata.png" width="300"/>
+
+#### **Container Page**
+
+The link in the task metadata sidebar takes you to the container page,
+which details the lifecycle of a container and their tasks. Like the
+host page, event logs exist charting the journey of a container task
+from initialization to termination. Task events such as container
+assignment and status changes are also recorded, as well as the clearing
+of a task from a container once it has run its course.
+
+![container_event_logs.png](../images/container_page.png)
+
+### Other Considerations
+
+### Disk Space
+
+Each instance is provisioned with 200GB of space; however, the actual
+disk space available for each container task can vary depending on the
+number and the nature of tasks sharing the same instance. In this sense,
+while CPU and memory are isolated allocations to each container, disk
+space is a shared resource across all containers on the same host. As
+such, dedicating more memory and CPU for a container makes it less
+likely to share the instance's disk space with other containers.
+Conversely, less resource-hungry containers are more likely to share the
+instance with others, so they will likely have a smaller share of the
+available disk space.
+
+In practical terms, this means that while each instance has a maximum of
+200GB of disk space, please bear in mind that the effective disk space
+available to your container tasks might be less and fluctuate, as other
+containers take and release disk space as they get created and exit,
+respectively. While we work on a more robust solution to this notion of
+isolating disk space, we recommend that you keep your container task
+disk space usage to a maximum of 10GB.
+
+### Git Cloning
+
+In a containerized environment, the hosts themselves are unable to be
+distributed with SSH keys used to clone GitHub repositories. This means
+that if git clones rely on the host's SSH keys to clone a private
+repository in your existing workflow, the cloning method will need to be
+changed to cloning via OAuth over HTTPS.
```

Comment:
Suggestion: rather than phrase this in terms of key distribution being impossible for Evergreen to do (we could do it if we were really intent on doing it), the reasoning is more due to the fact that it's not a good practice for everyone to share the same SSH key. This also likely only applies to modules, which specify the name of the repo to clone for the git URL.

We're already trying to get rid of SSH cloning, so I would add a mention to EVG-20972 that this section should also be deleted once SSH cloning is gone.